doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,258 @@
+"""
+============================================
+Restore spotted cornea image with inpainting
+============================================
+
+Optical coherence tomography (OCT) is a non-invasive imaging technique used by
+ophthalmologists to take pictures of the back of a patient's eye [1]_.
+When performing OCT,
+dust may stick to the reference mirror of the equipment, causing dark spots to
+appear on the images. The problem is that these dirt spots cover areas of
+in-vivo tissue, hence hiding data of interest. Our goal here is to restore
+(reconstruct) the hidden areas based on the pixels near their boundaries.
+
+This tutorial is adapted from an application shared by Jules Scholler [2]_.
+The images were acquired by Viacheslav Mazlin (see
+:func:`skimage.data.palisades_of_vogt`).
+
+.. [1] David Turbert, reviewed by Ninel Z Gregori, MD (2023)
+       `What Is Optical Coherence Tomography?
+       <https://www.aao.org/eye-health/treatments/what-is-optical-coherence-tomography>`_,
+       American Academy of Ophthalmology.
+.. [2] Jules Scholler (2019) "Image denoising using inpainting"
+       https://www.jscholler.com/2019-02-28-remove-dots/
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import plotly.io
+import plotly.express as px
+from scipy import sparse
+
+import skimage as ski
+
+
+#####################################################################
+# The dataset we are using in this example is an image sequence (a movie!) of
+# human in-vivo tissue. Specifically, it shows the *palisades of Vogt* of a
+# given cornea sample.
+
+#####################################################################
+# Load image data
+# ===============
+
+image_seq = ski.data.palisades_of_vogt()
+
+print(f'number of dimensions: {image_seq.ndim}')
+print(f'shape: {image_seq.shape}')
+print(f'dtype: {image_seq.dtype}')
+
+#####################################################################
+# The dataset is an image stack with 60 frames (time points) and 2 spatial
+# dimensions. Let us visualize 10 frames by sampling every six time points:
+# We can see some changes in illumination.
+# We take advantage of the ``animation_frame`` parameter in
+# Plotly's ``imshow`` function. As a side note, when the
+# ``binary_string`` parameter is set to ``True``, the image is
+# represented as grayscale.
+
+fig = px.imshow(
+    image_seq[::6, :, :],
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': '6-step time point'},
+    title='Sample of in-vivo human cornea'
+)
+plotly.io.show(fig)
```

Comment:
Right, I'm not sure how to control the relative sizes of the figure vs. frame... For me, it depends on the size of my web browser window. It doesn't clip anything if I make my window the size of my (14") screen -- without going *full screen* though.

Reply:
Setting the `height` and `width`parameter helps control the relative sizes of the figure vs. frame.

Reply:
> @decorouz hm, indirectly, yes, by controlling the _absolute_ height and width.
Yes, _absolute_

Reply:
After looking up [these docs](https://plotly.com/python/setting-graph-size/#setting-a-minimum-plot-size-with-automargins), I thought that 94ad574a4a3f would yield some improvement. It's hard for me to be confident about this because the same code isn't rendering the same way when I build the docs locally vs. CI.

Reply:
Ok, it's [looking](https://output.circle-artifacts.com/output/job/df2d6a3d-8254-41e6-8eda-5da76553f79e/artifacts/0/doc/build/html/auto_examples/applications/plot_cornea_spot_inpainting.html#sphx-glr-auto-examples-applications-plot-cornea-spot-inpainting-py)

* better and
* the same as locally

:smiley: 

The bottom tick labels are cropped out somehow, but I would argue it's acceptable: The info isn't lost since the `n` value (ranging 0-9) shows up with "6-step time point=`n`".